src/layers/core/polygon-layer/polygon-layer.js

```diff
@@ -111,9 +111,24 @@ export default class PolygonLayer extends CompositeLayer {
         getWidth,
         onHover,
         onClick,
-        updateTriggers: Object.assign({}, updateTriggers, {
+        updateTriggers: {
+          getWidth: updateTriggers.getWidth,
           getColor: updateTriggers.getColor
-        })
+        }
+      }));
+    } else if (extruded && wireframe && polygon) {
+      polygonOutlineLayer = new SolidPolygonLayer(Object.assign({},
```

Comment:
To me `outlineLayer` sounds like it is for the `Paths`, for clarity I think it could be worth a third variable - this would be the `polygonWireframeLayer`...